pkg/controller/keycloak/keycloak_controller.go

```diff
@@ -38,9 +38,9 @@ import (
 var log = logf.Log.WithName("controller_keycloak")
 
 const (
-	RequeueDelaySeconds      = 30 * time.Second
-	RequeueDelayErrorSeconds = 5 * time.Second
-	ControllerName           = "keycloak-controller"
+	RequeueDelay      = 30 * time.Second
```

Comment:
@slaskawi In this file the timeout parameters for controller were refactored. The question is why the backup file was not handled same way? https://github.com/keycloak/keycloak-operator/pull/208/files#diff-a7a5480b67785a9ee86a8ef0ecf17e19

Reply:
Could you please rephrase your question? I think I don't understand it... Maybe I'm a bit "slow" today...

Reply:
In this file it was renamed RequeueDelaySeconds to RequeueDelay and RequeueDelayErrorSeconds to RequeueDelayError.
But in this file it remained as it is: https://github.com/keycloak/keycloak-operator/pull/208/files#diff-a7a5480b67785a9ee86a8ef0ecf17e19

Reply:
ooh yes, you're probably right. @pb82 would you mind to issue a follow up PR?

Reply:
The linter complained about the constant name containing `seconds` when the value was already adjusted to seconds. This was not the case for the constant in the backup controller. Not sure how useful this particular linter rule is though.

Reply:
From what you're saying, code-wise we should be fine. However, it would be good to unify those variables.

Reply:
I've just created a PR: https://github.com/keycloak/keycloak-operator/pull/210